tests/e2e/utils/element-locators.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Returns common locators on the setup and settings page.
+ */
+export default class ElementLocators {
```

Comment:
Oh, I see why you are using this class in the specs. This approach works for now. As this file gets bigger, it would likely be better to move these locator helpers to methods on the relevant page object, like `page/settings.js` or `pages/setup.js`, and keep the common methods here, which can be imported into the relevant page helpers.